go.mod

```diff
@@ -84,10 +80,21 @@ require (
 	k8s.io/pod-security-admission v0.34.2
 	k8s.io/utils v0.0.0-20251002143259-bc988d571ff4
 	sigs.k8s.io/controller-runtime v0.22.4
-	sigs.k8s.io/controller-tools v0.19.0
 	sigs.k8s.io/yaml v1.6.0
 )
 
+tool (
+	github.com/ahmetb/gen-crd-api-reference-docs
+	github.com/ironcore-dev/vgopath
+	github.com/onsi/ginkgo/v2/ginkgo
+	go.uber.org/mock/mockgen
+	golang.org/x/tools/cmd/goimports
+	k8s.io/code-generator/cmd/go-to-protobuf
+	k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo
+	k8s.io/kube-openapi/cmd/openapi-gen
+	sigs.k8s.io/controller-tools/cmd/controller-gen
+)
```

Comment:
Previously, we specified versions for these dependencies. Which version will be pulled now? Always latest? Will this cause issues down the road?

Reply:
I see, thanks!